<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreerAuthaurisation</class>
 <widget class="QDialog" name="CreerAuthaurisation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>330</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>SizeVerCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Images/utilisateurs.png</normaloff>Images/utilisateurs.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#5D5D5D</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="global">
     <item row="0" column="0" rowspan="10">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>Images/utilisateurs.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="4">
      <widget class="QPushButton" name="bouton_annuler">
       <property name="styleSheet">
        <string notr="true">color:white;background-color:#5D5D5D;</string>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="4">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>25</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="text">
        <string>Ajouter un utilisateur à la base de données</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="4" colspan="3">
      <widget class="QPushButton" name="bouton_valider">
       <property name="styleSheet">
        <string notr="true">color:white;background-color:#5D5D5D</string>
       </property>
       <property name="text">
        <string>Valider</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="7">
      <widget class="QLabel" name="label_12">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="text">
        <string>    Une fois le nom entré une clef unique sera générée sous forme de QrCode. Vous aurez juste à ajouter cette clef aux différentes portes pour ajouter l'acces  </string>
       </property>
      </widget>
     </item>
     <item row="9" column="6">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="6">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="2" rowspan="2" colspan="4">
      <widget class="QLabel" name="label_error">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#FFBD00</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="6">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="1" rowspan="3" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <pointsize>17</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="text">
        <string>NOM Prénom</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="3" rowspan="3" colspan="3">
      <widget class="QLineEdit" name="edit_nom_prenom">
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
